controller/resume/impl.go

```diff
@@ -92,3 +92,29 @@ func (controller *ResumeController) ApproveResume(ctx *context.Context) error {
 
 	return ctx.JSON(http.StatusOK, struct{}{})
 }
+
+func (controller *ResumeController) DeleteResume(ctx *context.Context) error {
+	req := model.Resume{}
+
+	err := ctx.Bind(&req)
+	if err != nil {
+		return ctx.JSONError(
+			http.StatusBadRequest,
+			"Failed Bind",
+			"malformed request",
+			err,
+		)
+	}
+
+	err = controller.svc.Resume.DeleteResume(req.Username)
+	if err != nil {
+		return ctx.JSONError(
+			http.StatusBadRequest,
+			"Failed Resumes Approval",
+			"could not mark resume as approved",
```

Comment:
Error messages need to be updated, maybe something like this?
```suggestion
			"Failed Resumes Deletion",
			"could not delete resume",
```